charts/grand-challenge/templates/celery-beat-networkpolicy.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: celery-beat
+spec:
+  podSelector:
+    matchLabels:
+      service: celery-beat
+  policyTypes:
+    - Egress
+    - Ingress
+  ingress:
+  egress:
+    - to:
+        - podSelector:
+            matchLabels:
+              service: postgres
+      ports:
+        - protocol: TCP
+          port: 5432
```

Comment:
Any reason not to template the network policies?

Reply:
I may be missing something that should be templated, but in general, I try to template what's needed and not template what's no. Is there anything it seems I've missed here that you can spot?

Reply:
I see you have a different style compared to how the Cytomine helm chart is written. I don't spot anything but I was curious to know if there was any specific reason why didn't template the network policy ports and other stuff.

Reply:
Ah. Yes, to me, it's easier to read and work with if things aren't templated (unless it's needed/brings value).